index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio by Abenezer Samuel</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <h1 class="logo">Abenezer</h1>
+      <img src="./image/menu.png" alt="menu icon image" />
```

Comment:
- [ ] [REQUIRED] Good job so far! The `hamburger` button should have a `nav` tag. The `nav` tag is for semantics reasons and the right position for convention and user experience for mobile devices.